integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.service.ts

```diff
@@ -179,6 +190,10 @@ export class OpfCheckoutPaymentWrapperService {
   protected handlePaymentInitiationError(
     err: HttpErrorModel
   ): Observable<Error> {
+    if (isAuthorizationError(err)) {
+      throwError(err);
```

Comment:
i believe either we use
`return throwError(()=>err)`
or we use the non rxjs 'throw' without return:
`throw(()=>err`

below explanation:
https://rxjs.dev/api/index/function/throwError